### PR TITLE
Remove tile

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review

#118 
* adds functionality to remove a tile from the list of grid items

### CODE REVIEW

**Comment by nelsonic** [comment]
@jackcarlisle functionality *looks* good, please add a test so coverage does not decrease. thanks! 👍

**Comment by jackcarlisle** [comment]
@nelsonic will do!

File: src/assets/cancel.svg [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (lennym) [inline src/assets/cancel.svg]: Probably worth removing these `<g></g>`.

**Comment by jackcarlisle** [comment]
@lennym thanks will do!

**Comment by lennym** [comment]
My only thought is that this will likely cause some slightly odd behaviour once infinite scrolling is also brought in since I think it will it will mean that some search results won't be displayed if others are removed.

Example:

1) user loads results 0-9
2) user removes a result
3) user scrolls, triggering rendering of a slice of results starting from index 10 (see [code](https://github.com/numo-labs/isearch-ui/blob/24f0e25d271ad6866d5fea7fdaedfacd2b14df1b/src/components/isearch/index.js#L38))
4) since result 10 is now at index 9 on account of filtering, it is never rendered.

I'm not really sure if this is a problem.

**Comment by nelsonic** [comment]
@lennym good observation. would it be better to have a `visible` property for each tile and simply flip it to false instead of removing the item from the `state` ?

**Comment by lennym** [comment]
That makes sense as a solution, though I'd probably use a `hidden` flag rather than `visible` so we can default to visible without having to set the property everywhere.

**Comment by jackcarlisle** [comment]
@lennym good point!
@nelsonic I'll add the hidden property

**Comment by nelsonic** [comment]
@jackcarlisle is this good to go now then...?

**Comment by jackcarlisle** [comment]
@nelsonic yep!

**Comment by nelsonic** [comment]
@jackcarlisle sweet! 🍭

**Pull request merged**